test/parallel/test-worker-message-port.js

```diff
@@ -69,3 +69,12 @@ const { MessageChannel, MessagePort } = require('worker_threads');
     });
   });
 }
+
+{
+  assert.deepStrictEqual(
+    Object.getOwnPropertyNames(MessagePort.prototype).sort(),
+    [
+      'ref', 'unref', 'start', 'close', 'postMessage', 'onmessage',
+      'constructor'
+    ].sort());
```

Comment:
Nit: why should the array not be pre-sorted as in: `[ 'close',  'constructor', 'onmessage', 'postMessage',  'ref',  'start',  'unref' ]`?

Reply:
Laziness? ;) Fixed!